idunn/blocks/covid19.py

```diff
@@ -0,0 +1,40 @@
+from idunn import settings
+from .base import BaseBlock
+from .opening_hour import OpeningHourBlock
+from pydantic import constr
+from typing import ClassVar, List, Optional
+
+
+class Covid19Block(BaseBlock):
+    BLOCK_TYPE: ClassVar = "covid19"
+    STATUSES: ClassVar = ["open_as_usual", "open", "maybe_open", "closed", "unknown"]
+
+    status: constr(regex="({})".format("|".join(STATUSES)))
+    opening_hours: Optional[OpeningHourBlock]
+    note: Optional[str]
+
+    @classmethod
+    def from_es(cls, es_poi, lang):
+        if settings["BLOCK_COVID_ENABLED"] is not True:
+            return None
+        opening_hours = es_poi.properties.get("opening_hours:covid19")
+        status = "unknown"
+        if opening_hours == "same":
+            opening_hours = OpeningHourBlock.from_es(es_poi.properties.get("opening_hours"), lang)
+            status = "open_as_usual"
```

Comment:
`open_as_usual` should be used too if both tags `opening_hours:covid19` and `opening_hours` are equal